docs/content/en/main/concepts/investigations/edge-multi-cluster.md

```diff
@@ -0,0 +1,82 @@
+ requirements unlike those accomodated by upstream Kubernetes or KCP's Transparent MultiCluster (TMC). Ideally, we want declaritively defined workloads 
+ to operate across a flexible topology combining KCP workspaces and Kubernetes cluster namespaces to achieve greater scale and control over placement. 
+ Deployment of workloads across many thousands of destinations that include namespaced and non-namespaced objects, receiving state and reporting
+  status (grouped and individually), require new interfaces based on (and extending) KCP primitives.
+
+## Goal: Placement strategies
+
+There are many different ways to characterize deployments of workloads in the Edge use case. Support for blue/green, canary, carbon
+ intensity sensitive, and threat avoidance are some of the deployment strategies to consider when casting out 100's or 1000's of
+  workloads to millions of destinations. Having the ability to assign workloads using a declaritively defined placement strategy to
+   edge cluster locations, even if they are offline at times, is a key requirement of the edge multi-cluster use case.
+
+## Overlap of roles, responsibilities, and security
+
+Today's smartphone is not only the property of the user who purchased it. The reality is that application developers (app store),
+ corporate entities (vpn profiles), and the smartphone manufacturer have roles and responsibilities on these devices. Edge cluster
+  locations are no different in this respect. A scalable Edge multi-cluster solution can thrive and advance if there is support for
+   community contribution powered by integration and proper security features. All of these considerations must work across an evolving
+    hierarchy that combines small and large control plane API services.
+
+## Hierarchy and Scale are core considerations
+
+KCP has introduced the concept of a minimal API server which can be used to expand the number of places workloads can be deployed
+  by reducing the size of the control plane to a single binary. Furthermore, KCP's Transparent Multi-cluster (TMC) has made it possible to define
+  and deploy workloads centrally and then fan them out to specific edge locations. We would 
+   like to leverage different topologies (for instance: cascading multiple instances of KCP and Kubernetes cluster) to arrive at a 
+    distributed system capable of reaching any targeted edge cluster with/without edge devices attached. Topology flexibility 
+     will require a scalable database to hold the objects referenced by each of their associated API servers. The default database for 
+      KCP and Kubernetes is etcd. We will investigate alternatives to etcd distribution and replacement databases to increase 
+       scalability without sacrificing performance and resilience and considering factors which could negatively impact cost.
+
+## Areas of investigation
+
+1. Define the high level user Edge multicluster use case
```

Comment:
Maybe "Define the high level Edge multicluster use cases"  ?

Reply:
updated as suggested, thank you